docs/application/dotnet/guides/nui/animation-types.md

```diff
@@ -2,15 +2,15 @@
 
 ## Path Animation
 
-The `Path` class can be used to animate the position and orientation of actors. (a three-dimentional parametric curve)
+The `Path` class is three-dimensional parametric curve that is used to animate the position and orientation of actors.
```

Comment:
There is a just a missing article 'a', please make the small change:

The `Path` class is a three-dimensional parametric curve that is used to animate the position and orientation of actors.